study.md

```diff
@@ -25,7 +25,11 @@ In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+<!-- Sources: Pro Single Page Application Development Using Backbone.js and ASP.NET
+Authors: Gil Fink, Ido Flatow
+ISBN: 978-1-4302-6673-0 (Print) 978-1-4302-6674-7 (Online)
+
+SPA:It is a single page application which doesn't have major dependency on the server side. It is a single HTML page which allows the user to navigate through pages without long refreshes. -->
 ```
```

Comment:
Use any resources you'd like but please answer in your own words. It's okay if it's not quite and concise as your resource, but it's important for us to gauge an accurate understanding of how you understand it.